code/modules/vehicles/bike.dm

```diff
@@ -70,15 +70,15 @@
 	icon_state = "[bike_icon]_off"
 	if(storage_type)
 		storage_compartment = new storage_type(src)
-	if(!registration_plate)
+	if(!registration_plate && !organic)
```

Comment:
really the nicer way would be to refactor animal stuff to `/obj/vehicle/animal` or something, so they're not all subtypes of `bike`

Reply:
Refactored